frontend/src/pages/details-page.tsx

```diff
@@ -8,40 +8,42 @@ const DetailsPage = () => {
   const post: Post = state?.post;
   const navigate = useNavigate();
   return (
-    <div className="w-full font-[Poppins]">
-      <div className="relative flex flex-col">
-        <img
-          src={post.imageLink}
-          alt={post.title}
-          className="h-[360px] w-full object-cover md:h-[460px]"
-        />
-
-        <div className="absolute left-0 top-0 h-full w-full bg-black opacity-50"></div>
-        <div className="absolute top-16 w-full cursor-pointer justify-start px-4 text-lg text-white md:top-24 md:px-8 md:text-xl lg:px-16 lg:text-2xl">
+    <div className="dark:bg-dark-background min-h-screen">
+      <div className="dark:bg-dark-background min-h-min w-full font-[Poppins]">
+        <div className="relative flex flex-col">
           <img
-            src={navigateBackWhiteIcon}
-            style={{ height: 20, width: 40 }}
-            onClick={() => navigate(-1)}
+            src={post.imageLink}
+            alt={post.title}
+            className="h-[360px] w-full object-cover md:h-[460px]"
           />
```

Comment:
Kindly try using the tailwind-css recommended heights like h-48 which corresponds to 360 and 460 instead of arbitrary values

Reply:
@SankalpHaritash21 please resolve

Reply:
className="h-80 w-full object-cover md:h-96" 
@krishnaacharyaa  as h-80 is equivalent to 320px and h-96 is equivalent to 384px above this we need to give custom height

Reply:
className="h-80 w-full object-cover md:h-96"
@krishnaacharyaa as h-80 is equivalent to 320px and h-96 is equivalent to 384px above this we need to give custom height

Reply:
@krishnaacharyaa is this issue resolved